### ISSUE TITLE
Readme contains link to the dead fork of rust

### ISSUE THREAD

**Issue by Shchvova** [dvc94ch/riscv-rust-toolchain#29]
link https://github.com/riscv-rust/rust is broken

Problem is this is first search result which comes up when searching for "Risc-V rust" please, fix it, because it seems like rust on Risc-V is dead and this thing is obsolete.

```markdown
# OBSOLETE

This is no longer the recommended way to get a Rust toolchain for RISCV.
Instead, you can clone the [riscv-rust/rust](https://github.com/riscv-rust/rust)
fork of Rust and build it as normal:

    ./x.py build
```

**Comment by danc86** [comment]
Oops yeah, we better fix that. It's even better than that nowadays, you can just rustup a nightly and you get working RISCV support. :-)

**Comment by Shchvova** [comment]
Please do. I tried to find how to do it, but couldn’t. RISC-V not even on https://forge.rust-lang.org/platform-support.html

**Comment by Shchvova** [comment]
There’s rust issue https://github.com/rust-lang/rust/issues/34333 about risc rupport, but basically it stalled into “not working”.
I suggest updating documentation and links because from what I could find I concluded that rust doesn’t support risc.

**Comment by dvc94ch** [comment]
Please `rustup target add riscv32imac-unknown-none-elf`.

@danc86 Good thing I deleted the riscv-rust repo otherwise we'd be getting weekly reports from people trying to use it. Have you submitted a PR for advancing the rust lld fork? I think we can delete this repo too after to avoid confusion...

**Comment by danc86** [comment]
Yep lld in rust is updated on master, although I think it hasn't made it into any nightly build yet...

**Comment by laysakura** [comment]
@dvc94ch @danc86 Are there still any concerns about deleting this repo?

This repo is pinned to https://github.com/riscv-rust and it even seems like this repo is one of the most important for The RISC-V team.

**Comment by danc86** [comment]
I don't know if deleting the repo is a good idea, Github will let us mark the repo as "archived" which might be better. Then we won't break any existing historical links to the repo or old commits from it which might be referenced somewhere.

I think the README pretty clearly points people to the right place already, if they do happen to land here from a Google search or wherever.

Do you think we need to make the README _more_ obvious somehow, before we mark the repo as archived?

**Comment by laysakura** [comment]
It's comfortable for me just if this repo is unpinned from https://github.com/riscv-rust .

I assume pinned repositories are kind of most importantly maintained ones.

If I were a maintainer of this repo, I would unpin and archive it.
